lib/inferno/terminology/tasks/download_fhir_terminology.rb

```diff
@@ -19,8 +19,9 @@ def download_us_core
           FHIRPackageManager.get_package('hl7.fhir.us.core#4.0.0', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
           FHIRPackageManager.get_package('hl7.fhir.us.core#5.0.1', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
           FHIRPackageManager.get_package('hl7.fhir.us.core#6.1.0', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
+          FHIRPackageManager.get_package('hl7.fhir.us.core#7.0.0', PACKAGE_DIR, ['ValueSet', 'CodeSystem'])
           FHIRPackageManager.get_package(
-            'us.nlm.vsac#0.11.0', File.join(PACKAGE_DIR, 'vsac'), ['ValueSet', 'CodeSystem']
+            'us.nlm.vsac#0.17.0', File.join(PACKAGE_DIR, 'vsac'), ['ValueSet', 'CodeSystem']
```

Comment:
This is not the version of the VSAC package referenced in US Core 7.

Reply:
You'll also have to regenerate everything from scratch.

Reply:
Run the g10 tests and verify that things look normal compared with what's on prod.

Reply:
Re-ran with the vsac package update and everything looks normal and works as expected